src/main/java/com/marklogic/kafka/connect/sink/DefaultSinkRecordConverter.java

```diff
@@ -54,11 +54,17 @@ public class DefaultSinkRecordConverter implements SinkRecordConverter {
     private String mimeType;
     private boolean addTopicToCollections = false;
     private boolean includeKafkaMetadata = false;
+    private boolean includeKafkaHeaders = false;
+    private String kafkaHeadersPrefix = null;
```

Comment:
See how many of these can be made `final` - odds are both of these and a lot of the existing ones can all be `final`.